org.open_power.HardwareIsolation.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=open-POWER Host HardwareIsolation
```

Comment:
OpenPOWER

Reply:
Will change. My bad, I missed this to change (while testing I have seen that and thought to change)